README.md

```diff
@@ -87,142 +87,29 @@ Check the below video for a quick demo of XC3.
     git clone https://github.com/XgridInc/xc3.git
```

Comment:
I doubt we need to update pre_req as well. We are creating subnets in terraform that should not be pre_req and same key pair and IAM role we are taking care of it in init.sh. Correct me if I am wrong here

Reply:
Subnets are created using terraform.
key pair is created using the pre_req file.
IAM role, we are prompting user in the init.sh file.

Reply:
where is pre_req file execution command?
I suppose it should be removed from here then

Reply:
Removed from the readme.